### DISCUSSION TITLE
Is it possible to show some content in the node other than the port controls?

### DISCUSSION THREAD

**Discussion by idling-mind** [chrisjpatty/flume#124] [Q&A]
Hi,
  Thanks for this great library!
I've been toying with it for a while and was wondering if there is any way to show come content as part of the node body even if the input port has a connection? (for eg: the input or the output or combination of both rendered in some way)

**Comment by Perpaterb** [comment]
Are you talking about https://flume.dev/docs/controls#custom

inport a component and add it to the controls of the prot.

**Reply by idling-mind** [reply]
No, not really. I dont want the added content to be one of the ports (because it messes up with the input/output of the node).

**Comment by canxerian** [comment]
I would also like to use this feature
